adding_species_manuscript.tex

```diff
@@ -426,69 +426,68 @@ \section*{Introduction}\label{introduction}}
 
 Since its initial publication in \cite{Adrion2020}, several updates have been made
 to  the simulation framework of \stdpopsim and to its species catalog.
-One fundamental update was to upgrading the neutral simulation engine,
-\texttt{msprime}, from version 0.7.4 to version 1.0 \citep{Baumdicker2022}.
-This introduced a few important features, such as %
-
-\colorbox{yellow}{[ TODO: summarize key updates in msprime v1.0 in 1-2 sentences (Franz/Jerome?) ]}.
-
 
 Another key feature added to the simulation framework of \stdpopsim
 was modeling recombination by gene conversion. %
 \colorbox{yellow}{[ TODO: summarize model of GC in \stdpopsim in 5-7 sentences (Franz?) ]}
 
+% JK: Demoting this as it's a pretty minor change from a stdpopsim users'
+% perspective.
+In addition, the update to msprime version 1.0~\citep{Baumdicker2022}
+provides some benefits such as discrete sites mutations.
```

Comment:
perhaps "and opens the door to a future change to nucleotide-based mutation models"?

Reply:
Actually, we're already doing this. I'll update tomorrow.

Reply:
Okay, looks good. This is the only major new feature in msprime 1.0 that you think is worth highlighting (other than gene conversion)? I'll ping Franz to send a few sentences on gene conversion.